geomesa-bigtable/geomesa-bigtable-datastore/src/main/scala/org/locationtech/geomesa/bigtable/data/BigtableIndexAdapter.scala

```diff
@@ -27,37 +23,34 @@ class BigtableIndexAdapter(ds: BigtableDataStore) extends HBaseIndexAdapter(ds)
   override val tableNameLimit: Option[Int] = Some(50)
 
   override protected def configureScans(
-      originalRanges: Seq[Scan],
+      tables: Seq[TableName],
+      ranges: Seq[RowRange],
+      small: Boolean,
```

Comment:
What's the notion of 'small' here?

Reply:
in hbase parlance, gets are small and scans are not... although you can mark scans as 'small' too. i'm not entirely sure what effect it has in the end. we use it to decide if we use the multi-row-range filter or not.

Reply:
and we base it on whether we're doing single row lookups or range scans.